### PR TITLE
fix doc routing section inconsistency

### PR DESCRIPTION
**Pull request by pgnd**
fixing inconsistency in current 3.x docs 
_https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html_

### CODE REVIEW

File: docs/getting_started/installation.rst [hunk 1]
```diff
@@ -97,17 +97,6 @@ line in the `app/AppKernel.php` file of your project:
...
-
-    You must enable translator service in `config.yml`.
-
-    .. code-block:: yaml
-
-        framework:
-            translator: { fallbacks: ["%locale%"] }
-
-    For more information: http://symfony.com/doc/current/translation.html#configuration
-
```
Comment (greg0ire) [inline docs/getting_started/installation.rst]: Why did you remove that part?

Comment (pgnd) [reply docs/getting_started/installation.rst]: because it's also incosistent/redundant:

There's both
```
framework:
    translator: { fallbacks: ["%locale%"] }
```
and
```
framework:
    translator: { fallbacks: [en] }
```
Since I needed to bump the section # anyway, cleaned it up and stuck with the generic case

**Comment by jordisala1991** [comment]
Do we merge it with flintci complaining?

**Comment by greg0ire** [comment]
Yes, I checked the diff, it's unrelated
